docs/userguide/quickstart.rst

```diff
@@ -186,14 +186,12 @@ found, as shown in the example below:
     .. code-block:: ini
 
         [options]
-        packages = find: # OR `find_namespaces:` if you want to use namespaces
+        packages = find:  # OR `find_namespaces:` if you want to use namespaces
 
-        [options.packages.find] # (always `find` even if `find_namespaces:` was used before)
-        # This section is optional
-        # Each entry in this section is optional, and if not specified, the default values are:
```

Comment:
Is there a way of leaving some comments to make it clear that the entire section is optional, as well as every single parameter?

I think it would be beneficial, even if the information is redundant...